components/embl-logo/embl-logo.scss

```diff
@@ -12,13 +12,13 @@
 }
 
 @media (min-width: 500px) {
-  .embl-logo--responsive {
+  .embl-logo {
     background-image: url('../assets/embl-logo/assets/logo-medium.svg');
```

Comment:
Are we swapping out the SVG because of the rendering of the text?

What about moving the media queries into the SVG? So there's only one file if that's the case?

Reply:
Different SVGs show a different number of dots, and I think some manual spacing adjustments between dots.

I've not done mediaqueries in an SVG before, but I think that would mostly meen aggregating all three SVGs in one final and then displaying them conditionally ... so shouldn't be hard? My only real concern would be updating any minor tweaks from the design team ...

Maybe we add an issue to do this once it's more "stable"?